docs/docs/hive.md

```diff
@@ -126,9 +90,6 @@ To enable Hive support globally for an application, set `iceberg.engine.hive.ena
 For example, setting this in the `hive-site.xml` loaded by Spark will enable the storage handler for all tables created
 by Spark.
 
-!!! danger
-    Starting with Apache Iceberg `0.11.0`, when using Hive with Tez you also have to disable vectorization (`hive.vectorized.execution.enabled=false`).
```

Comment:
I looked at Hive4.0.x's support for Iceberg, and it seems to support vectorization. Correct me if I'm wrong.

Reply:
Hive4 supports both vectorized reads and writes on iceberg tables